Remove license comment for azurewebsites.net or umbraco.io (#5248)
umbraco/UmbracoDocs: 12/umbraco-commerce/getting-started/installation.md

Umbraco Commerce is fully functional during development, and whilst it is hosted on a local server (`localhost` or `.local` domains).

Hosting on any other public domain without a license will be limited to a maximum of 20 orders.

If you require an unrestricted staging environment, all licenses support two methods of allowing this:

* `staging.client.com` - Licenses allow unlimited subdomains, meaning you can host the staging site on a subdomain of the licensed domain.
* `clientcom.agency.com` - Licenses allow a concatenation of the licensed domain as a subdomain of any other domain.

> If you wish to host the site on any other URL such as a public dev / staging domain you want to access, please reach out to suits@umbraco.com for a test license.

Reviewer: Shortened the sentence a bit.

```suggestion
If you wish to host the site on any other URL like a public development domain, reach out to suits@umbraco.com for a test license.
```
